### DISCUSSION TITLE
How do you get @parcel/config-webextension to build an arbitrary HTML file?

### DISCUSSION THREAD

**Discussion by alexreg** [parcel-bundler/parcel#8831] [Q&A]
At the moment it's just ignored, since it's not referenced in `manifest.json`, and I don't believe there's a way to specify it.

Should I just be running the following?

```
parcel build src/manifest.json src/*.html
```

**Comment by mischnic** [comment]
Not sure what you expect to happen with these HTML pages (and how you want to use them), but yes that might behave like you want it to.

**Reply by alexreg** [reply]
@mischnic Just through chrome-extension URLs. Thanks for your reply and confirming, in any case. (Indeed, it seems the code is bundled correctly.)
